applications/ConvectionDiffusionApplication/tests/test_ConvectionDiffusionApplication.py

```diff
@@ -7,7 +7,7 @@
 import KratosMultiphysics.KratosUnittest as KratosUnittest
 
 try:
-    import KratosMultiphysics.ExternalSolversApplication as ExternalSolversApplication
+    import KratosMultiphysics.EigenSolversApplication as EigenSolversApplication
```

Comment:
better use Kratos_utilitis for this

also this check is not used, the tests that use the EigenSolversApp should only be run if it is available

Reply:
I did a search and replace. Let me check.

Reply:
Should be fine now.